pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -584,22 +587,19 @@ func (vs *VSphere) DisksAreAttached(volPaths []string, nodeName k8stypes.NodeNam
 		}
 		vm, err := vs.getVMByName(ctx, nodeName)
 		if err != nil {
+			if vclib.IsNotFound(err) {
+				glog.Infof("Node %q does not exist, all the disks on this node will be detached automatically.", nodeName)
```

Comment:
The current log statement is very confusing as when the node is not found, disks will not detached automatically. 
Can you rephrase this info log to
glog.Infof("Node %q does not exist, vsphere CP will assume all disks %v are not attached to it.", nodeName, volPaths).
Same at other places as well.